backend/src/db/providers/courseProvider.ts

```diff
@@ -152,13 +150,8 @@ const getCourseProvider = async (courseId: string | ObjectId, userId: string | O
   return populated;
 };
 
-const getMaterialsProvider = async ({ courseId, stage }: { courseId: string; stage?: string }) => {
-  const material = await CourseModel.findOne(
-    {
-      $and: [{ _id: courseId }, stage?.length ? { 'materials.stage': Number(stage) } : {}],
-    },
-    stage?.length ? { 'materials.$': 1 } : { materials: 1 },
-  ).lean();
+const getMaterialsProvider = async (courseId: string) => {
```

Comment:
how do we get materials for specific stage?

Reply:
Do we use it anywhere?